Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: down deploy shell up
+
+#setup image tag to latest for dev
+IMAGE_TAG?=latest
+DEPLOY_TYPE?=dev
+
+# stop containers
+down:
+	sudo docker compose down
+
+# run the deploy script, passing in deployment type
+deploy:
+	sudo ./deploy.sh -d $(DEPLOY_TYPE)
+
+# exec into api container
+shell:
+	sudo docker compose exec api sh
+
+# (local) development
+up:
+	IMAGE_TAG=$(IMAGE_TAG) sudo docker compose up --build -d
```

Comment:
1. Do we need sudo?
2. `IMAGE_TAG=$(IMAGE_TAG) sudo docker compose up` will pass `IMAGE_TAG` to `sudo` i.e. the first command, but will the env also be available to `docker compose up`? On my machine this doesn't appear to be so.

Reply:
I'll rm the sudo command on all this makefile commands - should be managed at host level. Docker installations have users added to the `docker` group or not (like angwin cluster)